src/uct/ib/rc/base/rc_iface.c

```diff
@@ -830,6 +1021,8 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_rc_iface_ops_t *ops, uct_md_h md,
     uct_rc_iface_tx_ops_cleanup(self);
 err_destroy_tx_mp:
     ucs_mpool_cleanup(&self->tx.mp, 1);
+err_destroy_dm:
```

Comment:
needs to be added before err_destroy_tx_mp

Reply:
fixed